### ISSUE TITLE
Where is the self.slot_decoder can be find?

### ISSUE THREAD

**Issue by BMM666hh** [Young1993/UGEN#3]
# encoder state
        hidden_states = encoder_outputs[0]
        slot_logits = self.slot_decoder(hidden_states)
        # slot_logits = slot_logits.masked_fill((1 - attention_mask.unsqueeze(1)).bool(), -10000.0)
        slot_outputs = torch.argmax(slot_logits, dim=2).cpu().detach().numpy().tolist()
        return slot_outputs
        
I find this code in models file  idsf.py. But I don't understand why the inference only use slot_outputs and  where is the self.slot_decoder can be find?
